staging/src/k8s.io/client-go/testing/fake.go

```diff
@@ -111,6 +111,8 @@ func (c *Fake) AddWatchReactor(resource string, reaction WatchReactionFunc) {
 
 // PrependWatchReactor adds a reactor to the beginning of the chain.
 func (c *Fake) PrependWatchReactor(resource string, reaction WatchReactionFunc) {
+	c.Lock()
+	defer c.Unlock()
```

Comment:
Please run the test TestAttacherMountDeviceWithInline 1000 tests to see if it is still flaky

bazel test --runs_per_test=1000 //pkg/volume/csi/...

Reply:
also what about other Methods, like AddWatchReactor above?

Reply:
I used the [stress](https://github.com/kubernetes/community/blob/4026287dc3a2d16762353b62ca2fe4b80682960a/contributors/devel/sig-testing/flaky-tests.md#deflaking-unit-tests) command for the unit test:

Here are my test commands:
```
go test ./pkg/volume/csi  -race -c

stress ./csi.test -test.run TestAttacherMountDeviceWithInline 
```

AddWatchReactor also updated.